server/src/actors/redis_actor.rs

```diff
@@ -0,0 +1,53 @@
+use std::collections::HashMap;
+
+use actix::prelude::*;
+use redis::{Client, Commands, Connection};
+
+use crate::models::messages::{SetScoreboardCommand, GetScoreboardCommand};
+
+#[derive(Debug)]
+pub struct RedisActor {
+    client: Client,
+}
+
+impl RedisActor {
```

Comment:
can we choose another struct name for more abstraction. like: DBActor, StoreActor,